src/utils/errorConstants.ts

```diff
@@ -0,0 +1,3 @@
+const GENERIC_ERROR_MESSAGE = "Something went wrong, please try again later.";
```

Comment:
Let's provide this one as an argument when initializing `ExceptionTransformer`. It is better to have it provided to the extractor dynamically. It may change from project to project and can be translated to multiple languages.